linux-tkg-config/prepare

```diff
@@ -797,68 +746,81 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\n
-      plain "0: No yield."
-      plain "1: Deboost and requeue task. (default)"
-      plain "2: Set rq skip task."
-      if [ "${_cpusched}" = "MuQSS" ]; then
-        read -rp "`echo $'\n      0. Supposedly best option for gaming performance,\n         but might lead to stability issues on some (AMD) platforms when combined with MuQSS\n    > 1. Default and recommended option for MuQSS - could lead to stability issues on some (Intel) platforms\n      2. Can be a good option with low rr_interval on MuQSS\n    [0-2?]: '`" CONDITION0;
-      else
-        read -rp "`echo $'\n    > 0. Recommended option for gaming on PDS - "tkg" default\n      1. Default, but can lead to stability issues on some platforms\n      2. Can be a good option with low rr_interval on MuQSS\n    [0-2?]: '`" CONDITION0;
+
+      _default_index="0"
+      if [ "${_cpusched}" = "muqss" ]; then
+        plain "For MuQSS"
+        yield_type_array_text=(
+          "No yield.\n        Supposedly best option for gaming performance but might lead to stability issues on some (AMD) platforms"
+          "Yield only to better priority/deadline tasks.\n        Could lead to stability issues on some (Intel) platforms"
+          "Expire timeslice and recalculate deadline.\n        Can be a good option with low rr_interval"
+        )
+      elif [[ "${_cpusched}" =~ ^(pds|upds)$ ]]; then
+        plain "For (U)PDS"
+        yield_type_array_text=(
+          "No yield.\n        Recommended option for gaming - \"tkg\" default"
+          "Yield only to better priority/deadline tasks.\n        Could lead to stability issues on some (Intel) platforms"
+          "Expire timeslice and recalculate deadline.\n        Can be a good option with low rr_interval"
+        )
+      else #BMQ
+        plain "For BMQ (experimental) - No recommended value yet, so try for yourself x) :"
+        yield_type_array_text=(
+          "No yield."
+          "Deboost and requeue task."
+          "Set rq skip task."
+        )
       fi
+
+      _prompt_from_array "${yield_type_array_text[@]}"
+      _sched_yield_type="${_selected_index}"
     fi
     if [ "${_cpusched}" = "upds" ] || ( [ "$_basever" = "54" ] || [ "$_basever" = "57" ] && [ "${_cpusched}" = "pds" ] ); then
       _sched="pds"
-    elif [ "${_cpusched}" = "MuQSS" ] || ( [ "$_basever" != "54" ] && [ "$_cpusched" != "bmq" ] && [ "$_cpusched" != "pds" ] ) || ( [ "$_basever" = "54" ] && [ "$_cpusched" = "bmq" ] ); then
+    elif [ "${_cpusched}" = "muqss" ] || ( [ "$_basever" != "54" ] && [ "$_cpusched" != "bmq" ] && [ "$_cpusched" != "pds" ] ) || ( [ "$_basever" = "54" ] && [ "$_cpusched" = "bmq" ] ); then
       _sched="${_cpusched}"
     else
       _sched="alt_core"
     fi
-    if [ "$CONDITION0" = "0" ]; then
+    if [ "$_sched_yield_type" = "0" ]; then
       sed -i -e 's/int sched_yield_type __read_mostly = 1;/int sched_yield_type __read_mostly = 0;/' ./kernel/sched/${_sched}.c
```

Comment:
For muqss, the file name is `MuQSS.c`. I don't see it handled here, which would lead to `muqss.c` -> not found. Did I miss it?

Reply:
No you did not, I just didn't notice this small thing that makes a huge difference. Will fix this

Reply:
It should be good now.

Reply:
No nevermind, I still need to fix few things